spec/features/posts_spec.rb

```diff
@@ -0,0 +1,125 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts' do
+  let!(:user) { FactoryBot.create(:user) }
+  let!(:post) { FactoryBot.create(:post, author: user) }
+  let!(:post2) { FactoryBot.create(:post, author: user, title: 'abc') }
+  let!(:comment) { FactoryBot.create(:comment, post:, text: '123') }
+  let!(:comment2) { FactoryBot.create(:comment, post:) }
+  let!(:comment3) { FactoryBot.create(:comment, post:) }
+  let!(:comment4) { FactoryBot.create(:comment, post:) }
+  let!(:comment5) { FactoryBot.create(:comment, post:) }
+  let!(:comment6) { FactoryBot.create(:comment, post:, text: 'holahola') }
+
+  describe 'Post #index' do
+    before do
+      visit user_posts_path(user)
+    end
+
+    it "I can see the user's profile picture." do
+      expect(page).to have_css("img[src*='']")
+    end
+
+    it "I can see the user's username." do
+      expect(page).to have_content(user.name)
+    end
+
+    it 'I can see the number of posts the user has written.' do
+      expect(page).to have_content("Number of posts: #{user.posts_counter}")
+    end
+
+    it "I can see a post's title." do
+      expect(page).to have_content(post.title)
+      expect(page).to have_content(post2.title)
+    end
+
+    it "I can see some of the post's body." do
+      expect(page).to have_content(post.text[0...20])
+      expect(page).to have_content(post2.text[0...20])
+    end
+
+    it 'I can see the first comments on a post.' do
+      expect(page).to have_content(comment6.text)
+      expect(page).to have_content(comment5.text)
+      expect(page).to have_content(comment4.text)
+      expect(page).to have_content(comment3.text)
+      expect(page).to have_content(comment2.text)
+      expect(page).to_not have_content(comment.text)
+    end
+
+    it 'I can see how many comments a post has.' do
+      expect(page).to have_content("Comments: #{post.comments_counter}")
+    end
+
+    it 'I can see how many likes a post has.' do
+      expect(page).to have_content("Likes: #{post.likes_counter}")
+    end
+
```

Comment:
- [ ] Good job so far, Kindly  add tests that asserts you can see a section for pagination if there are more posts than fit on the view 👍🏿